test/e2e/apps/job.go

```diff
@@ -879,3 +905,18 @@ func waitForJobFailure(ctx context.Context, c clientset.Interface, ns, jobName s
 		return false, nil
 	})
 }
+
+func sidecarContainer(name string) v1.Container {
+	containerRestartPolicyAlways := v1.ContainerRestartPolicyAlways
+
+	return v1.Container{
+		Name:  name,
+		Image: framework.BusyBoxImage,
+		Command: []string{
+			"sh",
+			"-c",
+			"sleep 100000",
```

Comment:
maybe just `sleep infinity`

Reply:
> sleep: invalid number 'infinity'

Actually, this image cannot recognize infinity.